internal/rbd/replication.go

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2021 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rbd
+
+import (
+	"context"
+	"strings"
+
+	librbd "github.com/ceph/go-ceph/rbd"
+	"github.com/csi-addons/spec/lib/go/replication"
```

Comment:
csi-addons  and grpc packages should not be imported. Where used, return an error on failure instead.

Reply:
This is still something that needs to be addressed. The cleanup is important so that `internal/rbd` does not use CSI-Addons functions/structs anymore.

Reply:
I can take this in a separate PR, @nixpanic, will that be fine?